raiden_contracts/utils/sign.py

```diff
@@ -105,12 +105,14 @@ def hash_reward_proof(
         chain_id,
         nonce):
     return Web3.soliditySha3([
+        'string',
         'uint256',
         'uint256',
         'address',
         'uint256',
         'uint256',
     ], [
+        "\x19Ethereum Signed Message:\n148",
```

Comment:
Would be good to reuse `eth_sign_hash_message`, to make sure the signature format used is the same (e.g. if we change it, we should do that in 1 place).